client/app/dbaas/logs/detail/streams/follow/streams-follow.service.js

```diff
@@ -0,0 +1,275 @@
+                        url
+                    }) });
+                } else {
+                    this.CloudMessage.success({ textHtml: this.$translate.instant("logs_streams_follow_copy_websocket_success") });
+                }
+            }
+        }
+
+        getTotalMessages () {
+            return this.totalMessages;
+        }
+
+        getLastEvent () {
+            return this.lastEvent;
+        }
+
+        getMessages () {
+            return this.messages;
+        }
+
+        getAlertType (level) {
+            return this.alertTypeLabelMap[level] ? this.alertTypeLabelMap[level].type : "";
+        }
+
+        getAlertLabel (level) {
+            return this.alertTypeLabelMap[level] ? this.alertTypeLabelMap[level].label : "";
+        }
+
+        isConnectionClosed () {
+            return this.connectionClosed;
+        }
+
+        isWaitingForMessages () {
+            return this.waitingForMessages;
+        }
+
+        /**
+         * Close websocket connection
+         */
+        closeConnection () {
+            this.webSocket.close();
+        }
+
+        /**
+         * Open websocket connection to given stream
+         * @param {object} stream
+         */
+        openConnection (stream) {
+            this.waitingForMessages = true;
+            this.connectionClosed = false;
+            this.messages = [];
+            this.totalMessages = 0;
+            this._connectToWebSocket(stream);
+        }
+
+        /**
+         * Copies GELF client command line to clipboard. Shows status message on UI.
+         * Shows error if copy failed, success otherwise.
+         * @param {Object} stream
+         * @param {string} gelfUrl
+         */
+        copyRFCCommandLine (stream, rfc5424Url) {
+            const token = this.LogsStreamsService.getStreamToken(stream);
+            if (token) {
+                let command = "";
+                const now = new Date();
+                const dateFormatted = now.toISOString();
+                command = `echo -e '<6>1 ${dateFormatted} 149.202.165.20 example.org - - [exampleSDID@8485 user_id="9001"  some_info="foo" some_metric_num="42.0" X-OVH-TOKEN="${token}"] A short RFC 5424 message that helps you identify what is going on\'\\n | openssl s_client -quiet -no_ign_eof  -connect ${rfc5424Url}`;
+                const error = this.ControllerHelper.copyToClipboard(command);
+                this._handleCommandCopyStatus(error, stream, command, this.testTypeEnum.RFC5424);
+            }
+        }
+
+        /**
+         * Copies GELF client command line to clipboard. Shows status message on UI.
+         * Shows error if copy failed, success otherwise.
+         * @param {Object} stream
+         * @param {string} gelfUrl
+         */
+        copyLTSVCommandLine (stream, ltsvUrl) {
+            const token = this.LogsStreamsService.getStreamToken(stream);
+            if (token) {
+                let command = "";
```

Comment:
let command = command = `echo -e 'X-OVH-TOKEN:${token}\\thost:example.org\\ttime:${dateFormatted}\\tmessage:A short LT...
It's a good practice to init variables as late as possibles when declaring them early doesn't provide extra readability / maintenability